Docs/userguide/quantization_workflow.rst

```diff
@@ -89,70 +39,68 @@ Supported precisions to run inference on target runtimes like |qnn|_ are:
 
 .. figure:: ../images/quantization_workflow_4.png
 
    Recommended quantization workflow
 
-FP16 precision (No quantization)
+The following sections describe the these processes.
+
+3. Trying FP16 precision (No quantization)
+------------------------------------------
+
+We recommend that you start by converting an FP32 model to FP16 precision without quantization. For details on how to compile FP16 models for target runtimes, see |qnn_docs|_ or |qai_hub_docs|_.
+
+4. Verifying W16A16 quantization
 --------------------------------
 
-Converting an FP32 model to FP16 precision without quantization is a recommended starting
-point. For more details on how to compile FP16 models for target runtimes, please refer to
-|qnn_docs|_ or |qai_hub_docs|_.
+Before using a quantized integer format, ensure that the FP32 model and the quantized model (QuantSim object) perform similarly during the forward pass, especially when custom quantizers are included in the model.
+
+.. note::
 
-W16A16 verification
--------------------
+  Ensure that the FP32 model adheres to model-specific guidelines. For instance, in PyTorch QuantSim can only quantize math operations performed by :class:`torch.nn.Module` objects, while :class:`torch.nn.functional` calls will be incorrectly ignored. Refer to framework-specific pages to learn more about such model guidelines.
 
-Before using quantized integer format, it's important to ensure that the FP32 model
-and the quantized model (QuantSim object) perform similarly during the forward pass, especially
-when custom quantizers are included in the model.
+Once the model conforms to guidelines, create a QuantSim with the bit-width set to 16 bits for both weights and activations (W16A16). 
 
-Set the bit-width to 16 bits for both weights and activations when creating the QuantSim.
-Then, obtain the off-target quantized accuracy metric for the quantized model and verify if
-it aligns with the FP32 model. If it doesn't, please report an issue to |aimet|_.
+Then, obtain the off-target quantized accuracy metric for the quantized model and verify that it agrees with the FP32 model. If it does not, you can help improve AIMET by reporting an issue to |aimet|_.
 
-Apply PTQ or QAT at specified precision
----------------------------------------
+5. Applying PTQ or QAT
+----------------------
 
-If any of the metrics are not acceptable with higher precision, begin with weights at
-INT8 precision and activations at INT16 precision. In this step, before creating the QuantSim,
-ensure that the FP32 model adheres to model specific guidelines. For instance, in PyTorch,
-QuantSim can only quantize math operations performed by :class:`torch.nn.Module` objects, while
-:class:`torch.nn.functional` calls will be incorrectly ignored. Please refer to framework specific
-pages to know more about such model guidelines.
+If any of the metrics are not acceptable with higher precision, begin with weights at INT8 precision and activations at INT16 precision. 
```

Comment:
We say "begin with" here, but didn't we actually begin with W16A16 in the previous step? Some more explanation might be a good idea here.